src/lib/desktop.ts

```diff
@@ -519,6 +525,60 @@ const getTokenFilePath = async () => {
   return window.electron.path.join(fullPath, TOKEN_FILE_NAME)
 }
 
+export const getEnvironmentConfigurationFolderPath = async () => {
+  const isTestEnv = window.electron.process.env.NODE_ENV === 'test'
+  const testSettingsPath = await window.electron.getAppTestProperty(
+    'TEST_SETTINGS_FILE_KEY'
+  )
+
+  const appConfig = await window.electron.getPath('appData')
+  const fullPath = isTestEnv
+    ? window.electron.path.resolve(testSettingsPath, '..')
+    : window.electron.path.join(
+        appConfig,
+        getAppFolderName(),
+        ENVIRONMENT_CONFIGURATION_FOLDER
+      )
+  return fullPath
+}
+
+export const getEnvironmentConfigurationPath = async (
+  environmentName: string
+) => {
+  const fullPath = await getEnvironmentConfigurationFolderPath()
+  try {
+    await window.electron.stat(fullPath)
+  } catch (e) {
+    // File/path doesn't exist
+    if (e === 'ENOENT') {
+      await window.electron.mkdir(fullPath, { recursive: true })
+    }
+  }
+  // /envs/development.json
```

Comment:
Is this comment correct? I though filenames were based on the domain.

Reply:
Good catch, thanks!